service-users/pom.xml

```diff
@@ -19,6 +19,14 @@
             <groupId>fi.nls.oskari.service</groupId>
             <artifactId>oskari-base</artifactId>
         </dependency>
+		<dependency>
```

Comment:
I don't think these are necessary anymore with the annotated UserContentService approach?